docs/source/getting_started/starting.rst

```diff
@@ -11,6 +11,15 @@ Start JupyterLab using:
 
 JupyterLab will open automatically in your browser.
 
+If the location of your notebook files are not in your home directory (more often the case in Windows, or if you have an external drive) you can use the `--notebook-dir` added parameter to start JupyterLab in that directory.  Be very careful though, never run it in your root volume (ex: `c/` in Windows, `/` in Linux) unless you know what you are doing and are prepared for the consequences! 
```

Comment:
The default command opens the current folder. So I would rephrase the sentence (+ taken into account the next comment).

```suggestion
If your notebook files are not in the current directory, you can pass your working directory path as argument when starting JupyterLab. Avoid running it from your root volume (e.g. `C:\` on Windows or `/` on Linux) to limit the risk of modifying system files.
```